src/m_input.c

```diff
@@ -702,6 +702,7 @@ void M_BindInputVariables(void)
 
     BIND_INPUT(input_menu_reloadlevel, "Restart current level/demo");
     BIND_INPUT(input_menu_nextlevel, "Go to next level");
+    BIND_INPUT(input_menu_prevlevel, "Go to previous level");
```

Comment:
It's config only? We don't have room for it in the "Key Bindings" menu. Maybe we should move player 1-4 to config:
![image](https://github.com/user-attachments/assets/5aa0a964-fba1-46f7-a87b-dfa1b65f197d)

Reply:
Yes, that's a good solution.